From fefdd79ebcd53a3035967789d004938ee39e2030 Mon Sep 17 00:00:00 2001
From: Carl George <carl@george.computer>
Date: Tue, 5 Apr 2022 18:49:13 -0500
Subject: [PATCH] BUILD(cmake): Find and link Poco::XML

Without this, building against the latest poco snapshot results in
errors like:

/usr/bin/ld: /builddir/build/BUILD/mumble-1.4.230.src/src/mumble/PluginManifest.cpp:72: undefined reference to `typeinfo for Poco::XML::Element'
---
 src/mumble/CMakeLists.txt | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/src/mumble/CMakeLists.txt b/src/mumble/CMakeLists.txt
index 159caeb2e..827c06a29 100644
--- a/src/mumble/CMakeLists.txt
+++ b/src/mumble/CMakeLists.txt
@@ -411,11 +411,16 @@ target_include_directories(mumble_client_object_lib
 		"${PLUGINS_DIR}"
 )
 
-find_pkg(Poco COMPONENTS Zip)
+find_pkg(Poco
+	COMPONENTS
+		XML
+		Zip
+)
 
 if(TARGET Poco::Zip)
 	target_link_libraries(mumble
 		PRIVATE
+			Poco::XML
 			Poco::Zip
 	)
 else()
-- 
2.35.1

